src/components/prendus-login/prendus-login.ts

```diff
@@ -158,11 +159,8 @@ async function signinUser(email: string, password: string, userToken: string | n
             }
         }
     `, {email, password}, userToken, (error: any) => {
-      const updatedError = {
-        ...error,
-        message: "Password or email information incorrect."
-      }
-      throw updatedError;
+      error.message = "No account found with that email and password combination"
```

Comment:
Hmm I don't recall doing this, might have been from a long time ago. I guess I could just create a message and then throw it instead of mutating the error?